scripts/ci-check.sh

```
@@ -850,14 +873,31 @@ run_all() {
+    fi
+
+    local lock_pid=""
+    lock_pid="$(cat "${lockfile}" 2>/dev/null || true)"
+    if [[ -n "${lock_pid}" ]] && kill -0 "${lock_pid}" 2>/dev/null; then
+      echo "error: ci-check.sh は既に実行中です (PID: ${lock_pid})" >&2
+      return 1
+    fi
+
+    rm -f "${lockfile}" >/dev/null 2>&1 || true
+  done
+  trap "rm -f '${lockfile}'" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)